src/dedicatedCloud/task/dedicatedCloud-task.v6.service.js

```diff
@@ -28,6 +28,29 @@ angular.module("ovh-api-services").service("OvhApiDedicatedCloudTaskV6", functio
                 state: "@state"
             }
         },
+        getGlobalTasks: {
```

Comment:
Prefer moving this to the `OvhApiDedicatedCloudV6` service instead.

see: https://github.com/ovh-ux/ovh-api-services/blob/master/src/dedicatedCloud/dedicatedCloud.v6.service.js#L40